client/homebrew/brewRenderer/brewRenderer.jsx

```diff
@@ -14,6 +14,9 @@ const NotificationPopup = require('./notificationPopup/notificationPopup.jsx');
 const Frame = require('react-frame-component').default;
 const dedent = require('dedent-tabs').default;
 
+const DOMPurify = require('dompurify');
+const purifyConfig = { FORCE_BODY: true, SANITIZE_DOM: false };
```

Comment:
`FORCE_BODY: true` - required to leave `<style>` tags in output
`SANITIZE_DOM: false` - required to leave manual IDs in output

If anyone can find a combination of other options to produce this functionality, we can switch to them.

From the DOMPurify source:
`FORCE_BODY` : "/* Decide if all elements (e.g. style, script) must be children of document.body. By default, browsers might move them to document.head */"
`SANITIZE_DOM` : "/* Output should be free from DOM clobbering attacks? This sanitizes markups named with colliding, clobberable built-in DOM APIs. */"

Enabling `FORCE_BODY` seems fine, but disabling `SANITIZE_DOM` seems like it might be excessive - I'm still trying to determine exactly what this parameter does.

Reply:
After further research, I believe that `SANITIZE_DOM: false` is the only way to allow manual IDs to persist, DOM clobbering removes the ability to set IDs as it can be an attack vector, but I don't believe that applies in our case.
Marking as resolved.